### ISSUE TITLE
hydroelastic: Links to SDFormat and URDF specification are broken?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [RobotLocomotion/drake#14107]
As of master @ 598ab9c303, I tried to click on the following links:
https://drake.mit.edu/doxygen_cxx/classdrake_1_1multibody_1_1_multibody_plant.html#mbp_hydroelastic_materials_properties
> ![image](https://user-images.githubusercontent.com/26719449/93915815-8bfc7380-fcd6-11ea-9f9d-7568d0fe43e2.png)

But those links appear to be broken; here's the HTML from the rendered Doxygen:
```
define them in an input URDF/SDF as detailed <a class="el" href="/media/ephemeral0/ubuntu/workspace/linux-bionic-gcc-bazel-nightly-documentation/src/doxygen_tmp/input/drake/multibody/parsing/detail_scene_graph.h.html#sdf_contact_material">here for SDF</a> or <a class="el" href="/media/ephemeral0/ubuntu/workspace/linux-bionic-gcc-bazel-nightly-documentation/src/doxygen_tmp/input/drake/multibody/parsing/detail_urdf_geometry.h.html#urdf_contact_material">here for URDF</a>.
```

**Comment by EricCousineau-TRI** [comment]
Here's that text in source:
https://github.com/RobotLocomotion/drake/blob/598ab9c303b9fb1782b86295c03af882fd098e4f/multibody/plant/multibody_plant.h#L1412-L1413

And here're the two anchors that are referred to:
https://github.com/RobotLocomotion/drake/blob/598ab9c303b9fb1782b86295c03af882fd098e4f/multibody/parsing/detail_urdf_geometry.h#L137
https://github.com/RobotLocomotion/drake/blob/598ab9c303b9fb1782b86295c03af882fd098e4f/multibody/parsing/detail_scene_graph.h#L124

FYI @SeanCurtis-TRI

**Comment by SeanCurtis-TRI** [comment]
I'm surprised they render as links. They end up linking into internal code. In my mind, this is part of #13314 (although it's not explicitly enumerated there).

**Comment by jwnimmer-tri** [comment]
Is this a duplicate of #14054?

**Comment by EricCousineau-TRI** [comment]
Ah, makes sense. Reassigning, as it's just a "What's documented publicly" question.

**Comment by EricCousineau-TRI** [comment]
@jwnimmer-tri Yup, seems so. Though this issue, with Sean's reference to #13314, I'd say the root-cause is a bit more clear?

**Comment by SeanCurtis-TRI** [comment]
This should get resolved by writing the user guide specified in #15796. I'm leaving it open until then.

**Comment by rpoyner-tri** [comment]
Another possibility is to link both of them to the drake:proximity_properties section in the parsing reference. Such a resolution would depend on #16444.
